examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs

```diff
@@ -629,8 +630,8 @@ public MutateOperation CreateRoot()
             MutateGoogleAdsResponse response = googleAdsServiceClient.Mutate(request);
 
             PrintResponseDetails(request, response);
-            // [END add_performance_max_product_listing_group_tree_1]
         }
+        // [END add_performance_max_product_listing_group_tree]
 
         // [START add_performance_max_product_listing_group_tree_7]
         /// <summary>
```

Comment:
Nit in [this comment](https://github.com/googleads/google-ads-dotnet/blob/6f0678260aa7913865f1c0aab24bd61ccaef1d06/examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs#L694) - should be "removed and/or created".